pandas/tests/groupby/test_transform.py

```diff
@@ -1074,3 +1074,34 @@ def test_transform_lambda_with_datetimetz():
         name="time",
     )
     assert_series_equal(result, expected)
+
+
+# @pytest.mark.parametrize(
+#     "input_df, expected_df",
+#     [
+#         (
+#             DataFrame(
+#                 {
+#                     "A": [121, 121, 121, 121, 231, 231, 676],
+#                     "B": [1, 2, np.nan, 3, 3, np.nan, 4],
+#                 }
+#             ),
+#             1,
+#         ),
+#         (DataFrame({"B": [6.0, 6.0, 6.0, 6.0, 3.0, 3.0, 4.0]}), 1),
+#     ],
+# )
+# def test_groupby_transform_sum(input_df, expected_df):
```

Comment:
Is this supposed to be commented out? We want to avoid having commented out code.